src/strategies/prepo-vesting/index.ts

```diff
@@ -6,9 +6,10 @@ export const author = 'prepo-io';
 export const version = '1.0.0';
 
 const abi = [
-  'function getAmountAllocated(address _recipient) external view override returns (uint256)',
-  'function getClaimableAmount(address _recipient) public view override returns (uint256)',
-  'function getVestedAmount(address _recipient) public view override returns (uint256)'
+  'function getAmountAllocated(address _recipient) external view returns (uint256)',
+  'function getClaimableAmount(address _recipient) public view returns (uint256)',
+  'function getVestedAmount(address _recipient) public view returns (uint256)',
```

Comment:
```suggestion
  'function getAmountAllocated(address recipient) external view returns (uint256)',
  'function getClaimableAmount(address recipient) external view returns (uint256)',
  'function getVestedAmount(address recipient) external view returns (uint256)',
```
see https://github.com/prepo-io/prepo-monorepo/blob/main/apps/smart-contracts/token/contracts/vesting/interfaces/IVesting.sol